modules/base/aphroditeAdapter.ts

```diff
@@ -0,0 +1,193 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
should we create an issue to update aphrodite?

Reply:
At least this version of aesthetic I don't think they (he) will care, since he's working on another version already and this is considered legacy.

Maybe when we allocate the effort of migrating to the new version there will be a better way, although I don't know if that version is ready already